src/app/shared/services/community.service.ts

```diff
@@ -1,21 +1,33 @@
-import { Communities } from '@shared/interfaces';
-import { HttpClient } from '@angular/common/http';
+import { Communities, Community } from '@shared/interfaces';
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
+import { AngularFirestore } from '@angular/fire/firestore';
+import { environment } from '@environments/environment';
 
 @Injectable({
   providedIn: 'root',
 })
 export class CommunityService {
   JSON_COMMUNITIES = 'assets/json/communities.json';
 
-  constructor(private httpClient: HttpClient) {}
+  constructor( public firestore: AngularFirestore) {
+
+  }
 
   get communities(): Observable<Communities> {
-    return this.httpClient
-      .get<Communities>(this.JSON_COMMUNITIES)
-      .pipe(map(communities => this.normalizeCommunities(communities)));
+    return this.firestore.collection(environment.firestore.COMMUNITIES_PATH).valueChanges().pipe(
```

Comment:
@Anexon, is it reactive? When a new inserting on firebase is performed, it emits a new value?

Reply:
@tonivj5 exactly

Reply:
@tonivj5 if you need the credential files of firebase and google storage let me know and I can help you getting them from the AngularCommunity, account.
As well as adding your email as admin to log in successfully

Reply:
Hey, yeah, how can I do it?

Reply:
It is a little bit tricky writing it down :D, would it be possible for you to do a quick hangout at some time today?

Reply:
Sure, I can :+1: